pkg/cluster/storage.go

```diff
@@ -66,9 +73,27 @@ func (c *Cluster) save(ctx context.Context) error {
 	return nil
 }
 
+// bootstrapKeyData retrieves and returns the data stored in
+// the datastore with the key "/bootstrap". This should only
+// exist with 1 entry and will return an error if more than
+// 1 entry is found.
```

Comment:
with appropriate line wrapping...
```suggestion
// bootstrapKeyData lists keys stored in the datastore with the prefix "/bootstrap", and will return the first such key. It will return an error if not exactly one key is found.
```

Reply:
Appropriate? Go makes no preference.

Reply:
I corrected the content, but was too lazy to wrap it lol